tools/ci/testrun/script/test_example/test_example.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/python3
+# encoding: utf-8
+import pytest
+
+pytestmark = [pytest.mark.common, pytest.mark.qemu]
+do_not_support = ["sabre-6quad", "rv-virt", "rv-virt64"]
```

Comment:
why not test c++

Reply:
> 

test on local c++ is works well on qemu, so enable on ci